pkg/agents/team3/foodCalculations.go

```diff
@@ -2,17 +2,99 @@ package team3
 
 import (
 	"math"
-
-	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/food"
 )
 
-/*
-stubbornness range = 0-100
-mood range = 0-100
-morality range = 0-100
-*/
+//To Do:
+//1. Remove int() from platFood once platFood is type int
+
+func foodReqCalc(a *CustomAgent3, targetHP int) int {
```

Comment:
you forgot to change this function. :)

Reply:
We decided not to change the helper functions within the foodCalculations because we dont think they would be member functions. We can do the changes if its necessary though.

Reply:
Given that they take in your specific agent only, it would be better for them to be member functions I think. But happy for you to change that in your next PR.